pkg/controllers/replica-controller/runner_test.go

```diff
@@ -249,14 +250,14 @@ func (r TestRunner) RunCommandWithLog(common string, args ...string) ([]byte, er
 
 // TestGetterProcess mocks zpool get.
 func TestGetterProcess(*testing.T) {
-	defer os.Exit(0)
+	defer syscall.Exit(0)
```

Comment:
has this been changed as per IDE suggestion? I believe some golang version had a different behaviour towards `os.Exit` being called from tests

Reply:
Yes.... that's why I'm using syscall. os.Exit() used in testing is an automatic failure.